lib/mods_display/fields/nested_related_item.rb

```diff
@@ -37,7 +37,15 @@ def to_html(view_context = ApplicationController.renderer)
 
     def related_item_mods_object(value)
       mods = ::Stanford::Mods::Record.new.tap do |r|
-        r.from_str("<mods xmlns=\"http://www.loc.gov/mods/v3\">#{value.children.to_xml}</mods>")
+        munged_node = value.dup.tap do |x|
+          # the mods gem expects the root of the document to have the tag <mods>
+          x.name = 'mods'
+
+          # nokogiri doesn't print out the namespaces unless we add at least one namespace
+          x.add_namespace_definition('ignoreme', 'http://example.com')
```

Comment:
@cbeer it's likely I'm not following something here, but the tests all still pass if I comment out this line. Is that expected?

Reply:
Huh. It looks like `value.dup` triggers the same magic as adding namespaces.. who knew 🤷‍♂️

Reply:
how odd